internal/command/jobs/printers.go

```diff
@@ -173,15 +165,15 @@ func getAttemptCreatedAtAndDuration(job *models.Job) (createdAtStr string, durat
 func getJobEnvironment(job *models.Job) string {
 	routingContext := job.Spec.RoutingContext
 
-	if routingContext == nil {
-		return "BASELINE"
-	}
-
-	if len(routingContext.Sandbox) > 0 {
-		return fmt.Sprintf("%s (SANDBOX)", routingContext.Sandbox)
+	switch {
+	case routingContext == nil:
+	case len(routingContext.Sandbox) > 0:
+		return fmt.Sprintf("sandbox=%s", routingContext.Sandbox)
+	case len(routingContext.Routegroup) > 0:
+		return fmt.Sprintf("routegroup=%s", routingContext.Routegroup)
 	}
 
-	return fmt.Sprintf("%s (ROUTEGROUP)", routingContext.Routegroup)
+	return ""
```

Comment:
Shouldn't this return `BASELINE`?

Reply:
To match UI yes, but we had empty in cli, @foxish any thoughts